### PR TITLE
Newsletter: Gutenberg Plugin: Add Send test email button

### PR DESCRIPTION
**Pull request by lezama**
## Proposed changes:

* Add Send test email button

Screenshot 2024-08-14 at 5 24 11 PM

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Add `&showNewsletterMenu` to the editor url

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ import { accessOptions } from '../../shared/memberships/constants';
 import { useAccessLevel } from '../../shared/memberships/edit';
 import illustration from './email-preview-illustration.svg';
 
-export default function EmailPreview( { isModalOpen, closeModal } ) {
+export function EmailPreview( { isModalOpen, closeModal } ) {
```
Comment (enejb) [inline projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js:30]: Do you want to export the function as well as the default here?

File: projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js [hunk 2]
```diff
@@ -48,18 +51,16 @@ const NewsletterMenu = () => {
...
-							variant="secondary"
-							disabled={ isPublished }
-						>
-							{ __( 'Preview email', 'jetpack' ) }
-						</Button>
+						<HStack>
+							<Button onClick={ openModal } variant="secondary" disabled={ isPublished }>
+								{ __( 'Preview email', 'jetpack' ) }
+							</Button>
+							<Button onClick={ openEmailPreview } variant="secondary" disabled={ isPublished }>
```
Comment (enejb) [inline projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js:58]: `openEmailPreview` seems to be confusing here. Should er rename this function to be be called `openSendTestEmail` or something like this?

**Comment by simison** [comment]
How does putting these side-by-side work when the labels are longer in some language?


image

This is probably rubbish but just trying :-D

File: projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js (hunk 2 above)
Comment (lezama) [reply projects/plugins/jetpack/extensions/blocks/subscriptions/menu.js:58]: good call, I am planning to do a refactor after I finish moving things around, noted.

File: projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js (hunk 1 above)
Comment (enejb) [reply projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js:30]: I see you are exporting multiple things!

**Review by enejb — DISMISSED** [review]
Looks good!

**Comment by lezama** [comment]
> How does putting these side-by-side work when the labels are longer in some language?

good call:

Screenshot 2024-08-14 at 6 34 51 PM

**Review by enejb — APPROVED** [review]
Looks good!

**Pull request merged**